rosa_learning/deploying_application_workshop/learning-lab-overview.adoc

```diff
@@ -0,0 +1,301 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="learning-lab-overview"]
+= Workshop overview
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: learning-lab-overview
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 22-JAN-2024
+//Modified for HCP 15 October 2024
+
+== Introduction
+
+After successfully provisioning your cluster, follow this workshop to deploy an application on it to understand the concepts of deploying and operating container-based applications. 
```

Comment:
I wonder if it this sentence we should specify the type of application that allows the user to understand the concepts? Not sure at this point in the reading if any kind of application would be used for learning concepts...could be just unclear on my part...